example/spec/parallel/cnmWorkflow/KinesisTestTriggerSpec.js

```diff
@@ -194,7 +194,7 @@ describe('The Cloud Notification Mechanism Kinesis workflow', () => {
       granuleDuplicates: {},
       granules: [
         {
-          granuleId: granuleId,
+          granuleId: record.product.name,
```

Comment:
Note: I think this may break when 4084 is merged

Reply:
Why? record.product.name is the granuleId when granuleId is not uniquified.